modules/administration-guide/pages/provisioning-namespaces.adoc

```diff
@@ -0,0 +1,112 @@
+:_content-type: CONCEPT
+:navtitle: Provisioning {orch-namespace}s
+:keywords: administration guide, provisioning, {orch-namespace}
+:page-aliases: 
+
+[id="configuring-{orch-namespace}s_{context}"]
+= Provisioning {orch-namespace}s
+
+This procedure describes how to provision workpaces {orch-namespace}s in advance, rather than relying on automatic provisioning. 
+
+.Prerequisites
+
+* The workspace user cannot create namespaces on {orch-name}.
+
+.Procedure
+
+. Create one {orch-namespace} for each user, with following labels and annotations. Do not create multiple namespaces for a single user: it may lead to undefined behavior.
++
+[subs="+quotes,+attributes"]
+----
+metadata:
+  labels: <1>
+    app.kubernetes.io/part-of: che.eclipse.org
+    app.kubernetes.io/component: workspaces-namespace
+  annotations: <2>
+    che.eclipse.org/username: __<username>__ <3>
+----
+<1> `server.customCheProperties.CHE_INFRA_KUBERNETES_NAMESPACE_LABELS` property value
+<2> `server.customCheProperties.CHE_INFRA_KUBERNETES_NAMESPACE_ANNOTATIONS` property value
+<3> Workspace user username on {orch-name}
+
+
+. Ensure the workspace user has `admin` role privileges in the target namespace:
++
+[subs="+quotes,+attributes"]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: admin
+  namespace: __<namespace>__ <1>
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: admin
```

Comment:
should be changed to edit permissions. Basically `oc adm policy add-role-to-user edit ${USER}` should be enough here

Reply:
I couldn't find the `kubectl` variant for `oc adm policy add-role-to-user edit ${USER}` :/.